jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEditorUtil.kt

```diff
@@ -90,6 +92,17 @@ object CodeWhispererEditorUtil {
         )
     }
 
+    fun shouldSkipInvokingBasedOnRightContext(editor: Editor): Boolean {
+        val caretContext = runReadAction { CodeWhispererEditorUtil.extractCaretContext(editor) }
+        val rightContextLines = caretContext.rightFileContext.split(Regex("\r?\n"))
+        val rightContextCurrentLine = if (rightContextLines.isEmpty()) "" else rightContextLines[0]
+
+        return CodeWhispererUserGroupSettings.getInstance().getUserGroup() == CodeWhispererUserGroup.RightContext &&
+            rightContextCurrentLine.isNotEmpty() &&
+            !rightContextCurrentLine.startsWith(" ") &&
+            rightContextCurrentLine.trim() != ("}")
```

Comment:
to double check, so we only check the current line right context is stirctly equal to "}"?
rightContextCurrentLine.trim() != ("}")

So case rightContextCurrentLine = "};" won't apply to "skip" case right

Reply:
yes, we wanted to keep it simple for now and the only exception is single "}"